pkg/cmd/server/origin_master.go

```diff
@@ -83,10 +85,18 @@ func (cfg Config) BuildOriginMasterConfig() (*origin.MasterConfig, error) {
 		return nil, err
 	}
 
+	dnsAddr := cfg.DNSBindAddr
```

Comment:
this feels similar to the warnings you get when :80 and :443 can't be bound to, but those happen at start time. I would move this earlier (to a validate func) or later (to the start func). Doing it in buildconfig feels weird